README.md

```diff
@@ -15,6 +15,29 @@ Ontimize web module for applications theming through '*scss*' files loading .
     * `@import 'node_modules/ontimize-web-ngx-theming/ontimize-theme.scss';`
     * `@include ontimize-theme-styles($theme);`
 
-  Also, if you want to use compact styles for *'Ontimize Web'* components, you have to include the following lines (theme definition is mandatory, optionally you can add a angular material typography configuration):
+
+  Also, if you want to use compact styles for *'Ontimize Web'* components, you have to include the following lines (theme definition is mandatory, optionally you can add a typography configuration):
   * `@import 'node_modules/ontimize-web-ngx-theming/ontimize-theme-lite.scss';`
-  * `@include ontimize-theme-styles-lite($theme, $typography);`
+  * `@include ontimize-theme-styles-lite($theme);`
+
+
+  You can create a custom a typography configuration including following files:
+  * `// Define a custom typography config that overrides the font-family as well as the headlines and title levels.`
+  * `$custom-typography: mat-typography-config( $font-family: 'Nunito Sans', $headline: mat-typography-level(16px, 22px, 400), $title: mat-typography-level(14px, 22px, 500));`
+
+  * `// Define a custom typography config that overrides the header height, row height, header font size header and row of the tables`
```

Comment:
_"header font size header and row of the tables"_ I do not understand that sentence